NuRadioReco/modules/io/event_parser_factory.py

```diff
@@ -9,11 +9,16 @@ def scan_files_function(version_major, version_minor):
     specifying the first version that function is for
     """
     def scan_files_2_0(self, iF, current_byte):
+        
+        if self.__parse_detector:
```

Comment:
Variables starting with `__` are private, so this will not work. We can promote `__parse_detector` to `_parse_detector` (single underscore) in NuRadioRecoio to get around this?

Reply:
done